app/models/place.rb

```diff
@@ -54,18 +30,18 @@ class Place
   validate :has_both_lat_lng_overrides
   validates_with CannotEditPlaceDetailsUnlessNewestInactiveDataset, on: :update
 
-  index({ location: "2d", snac: 1, service_slug: 1, data_set_version: 1 }, background: true)
-  index(service_slug: 1, data_set_version: 1)
-
-  # Index to speed up the `needs_geocoding` and `with_geocoding_errors` scopes
-  index(
-    service_slug: 1,
-    data_set_version: 1,
-    geocode_error: 1,
-    location: 1,
-  )
-
-  index({ name: 1 }, background: true)
+  # index({ location: "2d", snac: 1, service_slug: 1, data_set_version: 1 }, background: true)
+  # index(service_slug: 1, data_set_version: 1)
+  #
+  # # Index to speed up the `needs_geocoding` and `with_geocoding_errors` scopes
+  # index(
+  #   service_slug: 1,
+  #   data_set_version: 1,
+  #   geocode_error: 1,
+  #   location: 1,
+  # )
+  #
+  # index({ name: 1 }, background: true)
 
```

Comment:
Is it safe to remove this comment if the code is not needed?

Reply:
Fixed